app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpFragment.kt

```diff
@@ -31,6 +34,11 @@ class NextUpFragment : Fragment() {
 		}
 
 		binding.fragmentNextUpButtons.apply {
+			duration = get<UserPreferences>()[UserPreferences.nextUpTimeout].toLong()
```

Comment:
I'm unsure about how I'm setting duration here. In `NextUpButtons` I don't set `duration` or `countdownTimerEnabled` as nullable.
The intention is for duration to be the default value defined in `NextUpButtons` if for some reason getting the user preference fails/returns null.

Reply:
If this part of the code fails the complete fragment wouldn't work (setters for callbacks are below this line)
current approach seems fine to me, set the duration first and then checking if it should disable